.github/workflows/main.yaml

```diff
@@ -20,5 +20,5 @@ jobs:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/flutter_package.yml@v1
     with:
       flutter_channel: stable
-      flutter_version: 3.7.0
+      flutter_version: 3.3.10
```

Comment:
Hm, not sure if I want to decrease the flutter version of the CI to lower than the current stable.

Alchesmist is failing on 3.7?

Reply:
Yeah, the alchemist is failing on 3.7, you can look at the logs [here](https://github.com/erickzanardo/nes_ui/actions/runs/4008759358/jobs/6883318358#step:8:17).


As long as the project [requires 3.3.10 ](https://github.com/erickzanardo/nes_ui/blob/main/pubspec.yaml#L8), I feel  3.3.10 is ok. 

3.7 was released 2 days ago, I think the alchemist team needs time to work on that...

It's up to you mate!

Reply:
Alright! Back to the 3.7.0 🎉